### DISCUSSION TITLE
🆕 Implementing New Dashboard design

### DISCUSSION THREAD

**Discussion by PiyushKalyanpy** [PiyushKalyanpy/GyanaGuru#108] [General]
Connected issue #48 

![image](https://github.com/PiyushKalyanpy/GyanaGuru/assets/79275157/53871231-0bc6-4a49-990f-531bb478a500)

**Comment by mani200307** [comment]
I could work on with designing the **right side bar** for User details

**Reply by mani200307** [reply]
But, @PiyushKalyanpy there are duplicate issues as well #104 it is.

**Comment by kirtiraj22** [comment]
I can work on the sidebar menu.

**Comment by kirtiraj22** [comment]
hey @PiyushKalyanpy I have added the sidebar menu and have created a PR at #119, kindly check it and let me know if any further changes are required.

**Comment by An525ish** [comment]
Hey guys...I was assigned to change the appearance of the cards of "why choose us " section
--I hope it's going with the theme...If anyone want to suggest any changes they can

https://github.com/PiyushKalyanpy/GyanaGuru/assets/93612051/1e9745e4-8966-43ea-9b44-5079619e3b87
